web_autoflip_demo/video_upload/video_upload.ts

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2020 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. 
+*/
+
+//check if browser supports file api and filereader features
+if (window.File && window.FileReader && window.FileList && window.Blob) {
+  // this function is called when the input loads a video
+  function showVideo(): void {
+    var file = (<HTMLInputElement>event.target).files[0];
+    //grab the first video in the fileList
+    //Currently, only loading one file
+    console.log("video file has been chosen");
+    console.log(file);
+    renderVideo(file);
+  }
+
+  // this function is for displaying video content
+  function renderVideo(file: any): void {
+    var reader = new FileReader();
+    reader.onload = function (event) {
```

Comment:
I believe the google style guide calls for to put the type information, so replace "function (event) {" with "function (event: Event): void {". It's best to be explicit.

Reply:
Thank you so much for teaching me the google style guide. For this line, I realized that I don't need a event parameter. So I removed event and get the URL from the reader. Please check the new commit.